src/VirtoCommerce.SimpleExportImportModule.Web/Scripts/blades/file-upload.js

```diff
@@ -110,13 +110,12 @@ angular.module('virtoCommerce.simpleExportImportModule')
             $timeout(() => $document[0].querySelector('#selectPriceCsv').click());
         }
 
-        $scope.deleteUploadedItem = () => {
-            assetsApi.remove({urls: [blade.csvFileUrl]},
-                () => { },
-                (error) => bladeNavigationService.setError('Error ' + error.status, blade)
-            );
-
-            resetState();
+        $scope.deleteUploadedItem = (showConfirmation) => {
```

Comment:
Are there cases when confirmation wouldn't show?

Reply:
@pushnitsa When we are closing the 'file-upload' blade itself, without pressing delete button

Reply:
resolved, we'll show confirmation dialog for every scenario